.github/workflows/create-gitHub-prerelease-after-fast-forward-check.yml

```diff
@@ -0,0 +1,48 @@
+name: Create GitHub prerelease after fast-forward check
+on: workflow_dispatch
+
+jobs:
+  prerelease:
+    name: prerelease
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Access to the incoming release
+        uses: actions/checkout@v3
+        with:
+          repository: amzn/ion-java
+          path: current_release
+
+      - name: Build ion-java
+        run: |
+          cd current_release
+          git submodule init
+          git submodule update
+          mvn clean install
+
+      - name: Get the parameters of executable jar
+        run: |
+          cd current_release/target
+          echo "path=$(readlink -f $(ls *.jar))" >> $GITHUB_ENV
+          echo "file_name=$(ls *.jar)" >> $GITHUB_ENV
+          echo "version_number=$(echo $(ls *.jar) | cut -d'-' -f3)" >> $GITHUB_ENV
+
+      - name: Create new release
+        id: create_new_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: v${{ env.version_number }}
+          release_name: v${{ env.version_number }}
+          prerelease: true
```

Comment:
Can we verify in this workflow that no release for this version already exists? That would ensure that we don't accidentally create a new release without updating the version.

Reply:
> Can we verify in this workflow that no release for this version already exists? That would ensure that we don't accidentally create a new release without updating the version.
This workflow will be triggered manually after the version bump PR merged, but I agree. It would be a good idea to add one step to check the version number change in case this workflow is triggered accidentally.